modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/init-ext.jsp

```diff
@@ -0,0 +1,6 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
```

Comment:
We don't need to commit this file, there is nothing in it.

Reply:
I've removed it. There are plenty of similar ones around, couldn't it be that they have some internal function?

Reply:
I'm quite sure they have no internal function, they are just leftover junk.